### ISSUE TITLE
Limitation: the notifications are sent with the same number

### ISSUE THREAD

**Issue by msavazzi** [t0mer/green-api-custom-notifier#10]
if I'm not wrong you should add in the description:
When using your own number to send the notifications it will act like your are sending messages to yourself. that way no alert will popup. Consider using other phone number for that.

in the limitation section...
 :)

or at least this is what happens to me

**Comment by rowdynl** [comment]
Yeah, and that is why I bought a prepaid e-sim for 5 euros so I had an 'extra' number for my notifications. 

> if I'm not wrong you should add in the description: When using your own number to send the notifications it will act like your are sending messages to yourself. that way no alert will popup. Consider using other phone number for that.
> 
> in the limitation section... :)
> 
> or at least this is what happens to me
